src/fe-text/terminfo-core.c

```diff
@@ -535,6 +537,11 @@ void terminfo_cont(TERM_REC *term)
 	if (term->TI_smkx)
 		tput(tparm(term->TI_smkx));
 
+	if (bracketed_paste_enabled) {
+		bracketed_paste_enabled = FALSE;
+		term_set_bracketed_paste_mode(TRUE);
```

Comment:
`term_set_bracketed_paste_mode(bracketed_paste_enabled)` on CONT,
`term_set_bracketed_paste_mode(FALSE)` on stop, have `term_set_bracketed_paste_mode` only send the escape sequences, move `bracketed_paste_enabled` into `TERM_REC` and profit ?